api/hooks/customGameHook/index.js

```diff
@@ -30,24 +30,37 @@ module.exports = function gameHook() {
     },
     findOpenGames: function () {
       return new Promise(function (resolve, reject) {
-        const recentUpdateThreshhold = dayjs.utc().subtract(1, 'day')
+        const recentUpdateThreshold = dayjs.utc().subtract(1, 'day')
           .toDate();
+    
         Game.find({
           status: gameService.GameStatus.CREATED,
-          createdAt: { '>=': recentUpdateThreshhold },
+          createdAt: { '>=': recentUpdateThreshold },
         })
-          .populate('players')
+          .populate('players') // Populate the players array
           .exec(function (error, games) {
             if (error) {
               return reject(error);
-            } else if (!games) {
+            } else if (!games || games.length === 0) {
```

Comment:
We can combine these with optional chaining and utilize the fact that `0` is falsey like so
```suggestion
            } else if (!games?.length) {
```